tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -99,6 +151,22 @@ def scalars_impl(self, tag, run, output_format):
     else:
       return (values, 'application/json')
 
+  def _get_value(self, scalar_data_blob, dtype_enum):
+    """Obtains the value for a scalar event given the blob and dtype enum.
```

Comment:
Nit: `Obtains value for scalar event given blob and dtype enum.`

Reply:
Done. Leaving out articles is a good idea.